### PR TITLE
add extra dimension support

### PR DESCRIPTION
**Pull request by SophieGarden**
1 added support to creating indexes of flexible dimensions, AOAI embedding 3 large/small support an extra parameter called dimension
2 add a notebook to create multiple index in batch automatically

### CODE REVIEW

**Comment by SophieGarden** [comment]
do I need to remove our dataset and my path information in the notebooks? @ashatabak786

File: scripts/config.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-        "index_name": "<index name to use or create>",
-        "chunk_size": 1024,
-        "token_overlap": 128,
-        "semantic_config_name": "default",
-        "language": "<Language to support for example use 'en' for English. Checked supported languages here under lucene - https://learn.microsoft.com/en-us/azure/search/index-add-language-analyzers"
-    }
+[
+    {
+        "data_path": "<path to data>",
+        "location": "<azure region, e.g. 'westus2'>",
```
Comment (ashatabak786) [inline scripts/config.json:4]: what is the change here - i am unclear

Comment (SophieGarden) [reply scripts/config.json:4]: there is no change, not sure what happened here.

File: scripts/run_batch_create_index.ipynb
[the remark this answers is not in the document]
Comment (SophieGarden) [reply scripts/run_batch_create_index.ipynb:1]: Iconverted a similar batch run files to .py in DataGen, the thing is you always need to make changes for new cases...and notebook is much easier to debug.

[the remark this answers is not in the document]
Comment (SophieGarden) [reply scripts/run_batch_create_index.ipynb:1]: you can use this view to review https://github.com/microsoft/sample-app-aoai-chatGPT/blob/24d18de1c35b0b1bedc6f5464493a365b9d3b878/scripts/run_batch_create_index.ipynb

[the remark this answers is not in the document]
Comment (SophieGarden) [reply scripts/run_batch_create_index.ipynb:1]: converted to py script @minjieli99  please review again thanks!

**Review by ashatabak786 — APPROVED** [review]

**Pull request merged**